.semaphore/update_build_test.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2019 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+export LC_ALL=C.UTF-8
+
+sudo apt-get update
+sudo apt-get install --no-install-recommends --no-upgrade -qq $PACKAGES
```

Comment:
I'd prefer if this used the `./ci/` system from #16582.

Otherwise we'd probably lose a lot of coverage, like the wine tests, when switching from travis to semaphore